setup.py

```diff
@@ -1,15 +1,9 @@
 import os
-
 from setuptools import setup, find_packages
 
-version = '2.2.5+zg1.1'
-
-# TODO: once this branch is merged or in pip use, remove this
-os.system(
-    "pip3 install 'git+https://github.com/zillow/pipelines@feature/zg#egg=kfp&subdirectory=sdk/python'"
-)
+version = '2.2.5'
```

Comment:
Should this be upgraded to `2.3.0` (a minor change) since it looks like a lot of these Python upgrades might not be breaking but is a large functional change.
If there is going to be a lot of code breaking, perhaps a major change is warranted.

Reply:
I figured this was just the version from the upstream `metaflow` version, so I removed the `zg` specific version number and added in the same versioning policy as the rest of our libraries (ie. different versions for feature branches). 

I do have a task to rethink how best to version feature branches and plan to release an RFC on that when I have some time to write down my thoughts on this!